crates/cheatnet/src/cheatcodes/spy_events.rs

```diff
@@ -48,8 +48,7 @@ impl CheatnetState {
             .map(|event| {
                 let mut flattened_event = vec![];
                 if spy_on.does_spy(event.from) {
-                    flattened_event.push(Felt252::from_bytes_be(event.from.0.key().bytes()));
-                    flattened_event.push(event.name.clone());
```

Comment:
Do we no longer need an event `name` here?

Reply:
Yup, it is automatically added to the `keys`